src/diffusion_for_multi_scale_molecular_dynamics/noisers/lattice_noiser.py

```diff
@@ -1,5 +1,22 @@
+from dataclasses import dataclass
+from typing import Tuple
+
 import torch
 
+from experiments.analysis.analytic_score.repaint.plot_repaint_analytical_score_trajectories import \
```

Comment:
This should not be imported. `experiments` is a very unstable module and should not contain "real" code. Is suspect this was imported by mistake.

Reply:
Indeed, this looks like an auto-complete gone wrong.